README.md

```diff
@@ -698,3 +698,42 @@ module.settings.changeEdition = function (options, callback) {
   return callback(error, response)
 }
 ```
+
+#### getMappingMetadata
+This function can be used to add mapping validation for connectors. This function will be called on refresh to get the mapping metadata for connector.
```

Comment:
getMappingMetadata function is used to add mapping validation for connectors to ensure certain business critical mappings are not removed/edited by a business user. This function is called on page refresh/load to get the mapping metadata for connector stack to make mappings non-editable or required (non-deletable) or both for a front end user.